src/api/VoteMonitor.Api.Ngo/Handlers/NgoAdminCommandsHandler.cs

```diff
@@ -8,34 +8,37 @@
 using Microsoft.Extensions.Logging;
 using VoteMonitor.Api.Core.Services;
 using VoteMonitor.Api.Ngo.Commands;
+using VoteMonitor.Api.Ngo.Models;
 using VoteMonitor.Entities;
 
 namespace VoteMonitor.Api.Ngo.Handlers
 {
-    public class NgoAdminCommandsHandler : IRequestHandler<CreateNgoAdmin, Result>
+    public class NgoAdminCommandsHandler : IRequestHandler<CreateNgoAdmin, Result<NgoAdminModel>>
         , IRequestHandler<UpdateNgoAdmin, Result>
         , IRequestHandler<DeleteNgoAdmin, Result>
 
     {
         private readonly VoteMonitorContext _context;
         private readonly IHashService _hashService;
         private readonly ILogger<NgoAdminCommandsHandler> _logger;
+        private readonly IMapper _mapper;
 
-        public NgoAdminCommandsHandler(ILogger<NgoAdminCommandsHandler> logger, VoteMonitorContext context, IHashService hashService)
+        public NgoAdminCommandsHandler(ILogger<NgoAdminCommandsHandler> logger, VoteMonitorContext context, IHashService hashService, IMapper mapper)
         {
             _logger = logger;
             _context = context;
             _hashService = hashService;
+            _mapper = mapper;
         }
 
-        public async Task<Result> Handle(CreateNgoAdmin request, CancellationToken cancellationToken)
+        public async Task<Result<NgoAdminModel>> Handle(CreateNgoAdmin request, CancellationToken cancellationToken)
         {
             try
             {
-                var maxNgoAdminId = await _context.NgoAdmins.MaxAsync(x => x.Id, cancellationToken);
+                var maxNgoAdminId = await _context.NgoAdmins.MaxAsync(x => (int?)x.Id, cancellationToken);
```

Comment:
I know is not related to your code, is more for me. Is there a good reason why an Identity column type was not used for Id?  The line 38 is not thread safe and can result in same number for 2 different calls.